server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/MIMEMessageConverterTest.java

```diff
@@ -406,7 +466,7 @@ public void convertShouldGenerateExpectedMultipartWhenHtmlAndTextBodyProvided()
 
         String expectedHeaders = "MIME-Version: 1.0\r\n" +
                 "Content-Type: multipart/alternative;\r\n" +
-                " boundary=\"-=Part.1.";
+                " boundary=\"-=Part.";
```

Comment:
is it related ?

Reply:
yes more or less, both are using MimeUtil which is using (between others) a static counter to get a unique id. The same counter is used for generating boundaries and message ids. Of course that's not the only thing that makes the id unique.

Reply:
Sorry to be picky, but that means the assertions is too strong and we should weaken it

Reply:
that's exactly what I did